contribute/design/1.x/jiva/2019152019-jiva-autosnap-deletion.md

```diff
@@ -0,0 +1,98 @@
+
+---
+oep-number: draft Auto Snapshot deltion 20190715
+title: Jiva auto snapshot deletion
+authors:
+  - "@utkarshmani1997"
+owners:
+  - "@vishnuitta"
+  - "@payes"
+  - "@utkarshmani1997"
+editor: "@utkarshmani1997"
+creation-date: 2019-07-15
+last-updated: 2019-07-15
+status: provisional
+see-also:
+  - NA
+replaces:
+  - NA
+superseded-by:
+  - NA
+---
+
+ # Jiva automatic snapshot deletion
+
+ ## Table of Contents
+
+ * [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+* [Proposal](#proposal)
+    * [User Stories](#user-stories)
+      * [Delete autogenerated snapshot](#delete-autogenerated-snapshot)
+    * [Implementation Details](#implementation-details)
+* [Drawbacks](#drawbacks)
+* [Alternatives](#alternatives)
+
+ ## Summary
+
+ This proposal is aimed for the users who experience very frequent network glitch
+ at there cluster, due to which replica get restarted and autogenerated snapshots
+ files are created. Currently the limit is 512 snapshots, once these no of snapshots
```

Comment:
_n: use the following instead of `Currently the limit is 512 snapshots,...`

```
A max of 512 snapshots can be created. Once 512 snapshots are created, the replica won't be able to connect again automatically and requires manual cleanup of snapshot.
```